src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,70 @@
+package step2.domain;
+
+import step2.exception.LottoException;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+    public static final int MIN_LOTTONUMBER = 1;
+    public static final int MAX_LOTTONUMBER = 45;
```

Comment:
- 해당 상수들은 `LottoNumber` 내부에서만 사용하니 접근 제어자를 `private`으로 선언해보면 어떨까요?
아니면 `LottoFactory`에서 해당 상수들을 활용해볼 수도 있을 것 같아요.
- 여기도 띄어쓰기를 추가하면 조금이나마 가독성이 좋아질 것 같습니다.
```suggestion
    public static final int MIN_LOTTO_NUMBER = 1;
    public static final int MAX_LOTTO_NUMBER = 45;
```